test/unit/spec/narrativeLoginSpec.js

```diff
@@ -69,14 +69,15 @@ define ([
                 response: [{}]
             });
             return Login.init($node, true)  // true here means there's no kernel
-                .then(() => {
+                .finally(() => {
```

Comment:
If UserMenu (which we're not testing, but gets run anyway) throws an error, it cleans up and returns anyway. Setting `finally` here makes things run more smoothly, though it can probably get changed back to `then`.